core/command/bundle.js

```diff
@@ -28,8 +28,8 @@ function builder (yargs) {
       output: {
         alias: 'o',
         type: 'string',
-        default: 'prepared',
-        describe: 'In which folder will the prepared presentation be written'
+        default: 'bundled',
```

Comment:
Maybe, add an alias to `prepare` for back-compatibility? Here you can add the field `alias: "prepare"`

Reply:
Sure, backward compatibility is always good. Though I’m not sure what this option does and this is going to help. Could you please give an example? Thank you.

Reply:
For example, I have several bash scripts in CI. After updating the CLI, you will need to carefully rename the commands in the scripts.